Request: Turn on VTK

Turning on VTK in the ants build enables more tools.

@gdevenyi - I attempted to build ANTS (current master) with `USE_VTK=ON`, but I run into a build error. Does VTK have to compiled before compiling ANTs? Can you list the necessary steps to build ANTs with VTK on?

Related to stnava/ANTs#310 and stnava/ANTs#355.

```
CMake Error at Modules/Bridge/VtkGlue/itk-module-init.cmake:9 (find_package):
  Could not find a package configuration file provided by "VTK" with any of
  the following names:

    VTKConfig.cmake
    vtk-config.cmake

  Add the installation prefix of "VTK" to CMAKE_PREFIX_PATH or set "VTK_DIR"
  to a directory containing one of the above files.  If "VTK" provides a
  separate development package or SDK, be sure it has been installed.
Call Stack (most recent call first):
  CMake/ITKModuleEnablement.cmake:316 (include)
  CMakeLists.txt:317 (include)


-- Configuring incomplete, errors occurred!
See also "/tmp/ANTs/build/ITKv4-build/CMakeFiles/CMakeOutput.log".
See also "/tmp/ANTs/build/ITKv4-build/CMakeFiles/CMakeError.log".
make[2]: *** [ITKv4-prefix/src/ITKv4-stamp/ITKv4-configure] Error 1
make[1]: *** [CMakeFiles/ITKv4.dir/all] Error 2
make: *** [all] Error 2
```

This is my log building with -j 1. If I build with -j 2 or higher then VTK will build but ITK fails with the same error.

```Cloning into 'ants'...
Note: checking out 'tags/v2.2.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 0740f91... ENH:  Add clarification.
-- The C compiler identification is GNU 5.4.0
-- The CXX compiler identification is GNU 5.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test C_HAS_WARNING-Wno-uninitialized
-- Performing Test C_HAS_WARNING-Wno-uninitialized - Success
-- Performing Test C_HAS_WARNING-Wno-unused-parameter
-- Performing Test C_HAS_WARNING-Wno-unused-parameter - Success
-- Performing Test C_HAS_WARNING-Wall
-- Performing Test C_HAS_WARNING-Wall - Success
-- Performing Test C_HAS_WARNING-Wcast-align
-- Performing Test C_HAS_WARNING-Wcast-align - Success
-- Performing Test C_HAS_WARNING-Wdisabled-optimization
-- Performing Test C_HAS_WARNING-Wdisabled-optimization - Success
-- Performing Test C_HAS_WARNING-Wextra
-- Performing Test C_HAS_WARNING-Wextra - Success
-- Performing Test C_HAS_WARNING-Wformat=2
-- Performing Test C_HAS_WARNING-Wformat=2 - Success
-- Performing Test C_HAS_WARNING-Winvalid-pch
-- Performing Test C_HAS_WARNING-Winvalid-pch - Success
-- Performing Test C_HAS_WARNING-Wno-format-nonliteral
-- Performing Test C_HAS_WARNING-Wno-format-nonliteral - Success
-- Performing Test C_HAS_WARNING-Wpointer-arith
-- Performing Test C_HAS_WARNING-Wpointer-arith - Success
-- Performing Test C_HAS_WARNING-Wshadow
-- Performing Test C_HAS_WARNING-Wshadow - Success
-- Performing Test C_HAS_WARNING-Wunused
-- Performing Test C_HAS_WARNING-Wunused - Success
-- Performing Test C_HAS_WARNING-Wwrite-strings
-- Performing Test C_HAS_WARNING-Wwrite-strings - Success
-- Performing Test C_HAS_WARNING-funit-at-a-time
-- Performing Test C_HAS_WARNING-funit-at-a-time - Success
-- Performing Test C_HAS_WARNING-Wno-strict-overflow
-- Performing Test C_HAS_WARNING-Wno-strict-overflow - Success
-- Performing Test CXX_HAS_WARNING-Wall
-- Performing Test CXX_HAS_WARNING-Wall - Success
-- Performing Test CXX_HAS_WARNING-Wcast-align
-- Performing Test CXX_HAS_WARNING-Wcast-align - Success
-- Performing Test CXX_HAS_WARNING-Wdisabled-optimization
-- Performing Test CXX_HAS_WARNING-Wdisabled-optimization - Success
-- Performing Test CXX_HAS_WARNING-Wextra
-- Performing Test CXX_HAS_WARNING-Wextra - Success
-- Performing Test CXX_HAS_WARNING-Wformat=2
-- Performing Test CXX_HAS_WARNING-Wformat=2 - Success
-- Performing Test CXX_HAS_WARNING-Winvalid-pch
-- Performing Test CXX_HAS_WARNING-Winvalid-pch - Success
-- Performing Test CXX_HAS_WARNING-Wno-format-nonliteral
-- Performing Test CXX_HAS_WARNING-Wno-format-nonliteral - Success
-- Performing Test CXX_HAS_WARNING-Wpointer-arith
-- Performing Test CXX_HAS_WARNING-Wpointer-arith - Success
-- Performing Test CXX_HAS_WARNING-Wshadow
-- Performing Test CXX_HAS_WARNING-Wshadow - Success
-- Performing Test CXX_HAS_WARNING-Wunused
-- Performing Test CXX_HAS_WARNING-Wunused - Success
-- Performing Test CXX_HAS_WARNING-Wwrite-strings
-- Performing Test CXX_HAS_WARNING-Wwrite-strings - Success
-- Performing Test CXX_HAS_WARNING-funit-at-a-time
-- Performing Test CXX_HAS_WARNING-funit-at-a-time - Success
-- Performing Test CXX_HAS_WARNING-Wno-strict-overflow
-- Performing Test CXX_HAS_WARNING-Wno-strict-overflow - Success
-- Performing Test CXX_HAS_WARNING-Wno-deprecated
-- Performing Test CXX_HAS_WARNING-Wno-deprecated - Success
-- Performing Test CXX_HAS_WARNING-Wno-invalid-offsetof
-- Performing Test CXX_HAS_WARNING-Wno-invalid-offsetof - Success
-- Performing Test CXX_HAS_WARNING-Woverloaded-virtual
-- Performing Test CXX_HAS_WARNING-Woverloaded-virtual - Success
-- Performing Test CXX_HAS_WARNING-Wstrict-null-sentinel
-- Performing Test CXX_HAS_WARNING-Wstrict-null-sentinel - Success
-- Performing Test CXX_HAS_DISABLE_OPTIMIZATION_FLAG
-- Performing Test CXX_HAS_DISABLE_OPTIMIZATION_FLAG - Failed
-- Found Git: /usr/bin/git (found version "2.7.4") 
-- SuperBuild - ANTS => Requires VTK, ITKv4, 
-- SuperBuild -   VTK[OK]
-- SuperBuild -   ITKv4[OK]
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.8") 
-- SuperBuild - ANTS[OK]
-- Configuring done
-- Generating done
-- Build files have been written to: /opt/builds/ants-build
Scanning dependencies of target ITKv4
[  4%] Creating directories for 'ITKv4'
[  8%] Performing download step (git clone) for 'ITKv4'
Cloning into 'ITKv4'...
Note: checking out '5aad8ac86968a12be74a31ef6493811c0ea13d05'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 5aad8ac... BUG: Add missing ZLib deflate support to HDF5
[ 12%] No patch step for 'ITKv4'
[ 16%] No update step for 'ITKv4'
[ 20%] Performing configure step for 'ITKv4'
Not searching for unused variables given on the command line.
-- The C compiler identification is GNU 5.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- The CXX compiler identification is GNU 5.4.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Git: /usr/bin/git (found version "2.7.4") 
-- Could NOT find PythonInterp (missing:  PYTHON_EXECUTABLE) 
-- Could NOT find PythonInterp (missing:  PYTHON_EXECUTABLE) 
-- Performing Test C_HAS_WARNING-Wno-uninitialized
-- Performing Test C_HAS_WARNING-Wno-uninitialized - Success
-- Performing Test C_HAS_WARNING-Wno-unused-parameter
-- Performing Test C_HAS_WARNING-Wno-unused-parameter - Success
-- Performing Test C_HAS_WARNING-Wall
-- Performing Test C_HAS_WARNING-Wall - Success
-- Performing Test C_HAS_WARNING-Wno-long-double
-- Performing Test C_HAS_WARNING-Wno-long-double - Failed
-- Performing Test C_HAS_WARNING-Wcast-align
-- Performing Test C_HAS_WARNING-Wcast-align - Success
-- Performing Test C_HAS_WARNING-Wdisabled-optimization
-- Performing Test C_HAS_WARNING-Wdisabled-optimization - Success
-- Performing Test C_HAS_WARNING-Wextra
-- Performing Test C_HAS_WARNING-Wextra - Success
-- Performing Test C_HAS_WARNING-Wformat_2
-- Performing Test C_HAS_WARNING-Wformat_2 - Success
-- Performing Test C_HAS_WARNING-Winvalid-pch
-- Performing Test C_HAS_WARNING-Winvalid-pch - Success
-- Performing Test C_HAS_WARNING-Wno-format-nonliteral
-- Performing Test C_HAS_WARNING-Wno-format-nonliteral - Success
-- Performing Test C_HAS_WARNING-Wpointer-arith
-- Performing Test C_HAS_WARNING-Wpointer-arith - Success
-- Performing Test C_HAS_WARNING-Wshadow
-- Performing Test C_HAS_WARNING-Wshadow - Success
-- Performing Test C_HAS_WARNING-Wunused
-- Performing Test C_HAS_WARNING-Wunused - Success
-- Performing Test C_HAS_WARNING-Wwrite-strings
-- Performing Test C_HAS_WARNING-Wwrite-strings - Success
-- Performing Test C_HAS_WARNING-funit-at-a-time
-- Performing Test C_HAS_WARNING-funit-at-a-time - Success
-- Performing Test C_HAS_WARNING-Wno-strict-overflow
-- Performing Test C_HAS_WARNING-Wno-strict-overflow - Success
-- Performing Test CXX_HAS_WARNING-Wall
-- Performing Test CXX_HAS_WARNING-Wall - Success
-- Performing Test CXX_HAS_WARNING-Wno-long-double
-- Performing Test CXX_HAS_WARNING-Wno-long-double - Failed
-- Performing Test CXX_HAS_WARNING-Wcast-align
-- Performing Test CXX_HAS_WARNING-Wcast-align - Success
-- Performing Test CXX_HAS_WARNING-Wdisabled-optimization
-- Performing Test CXX_HAS_WARNING-Wdisabled-optimization - Success
-- Performing Test CXX_HAS_WARNING-Wextra
-- Performing Test CXX_HAS_WARNING-Wextra - Success
-- Performing Test CXX_HAS_WARNING-Wformat_2
-- Performing Test CXX_HAS_WARNING-Wformat_2 - Success
-- Performing Test CXX_HAS_WARNING-Winvalid-pch
-- Performing Test CXX_HAS_WARNING-Winvalid-pch - Success
-- Performing Test CXX_HAS_WARNING-Wno-format-nonliteral
-- Performing Test CXX_HAS_WARNING-Wno-format-nonliteral - Success
-- Performing Test CXX_HAS_WARNING-Wpointer-arith
-- Performing Test CXX_HAS_WARNING-Wpointer-arith - Success
-- Performing Test CXX_HAS_WARNING-Wshadow
-- Performing Test CXX_HAS_WARNING-Wshadow - Success
-- Performing Test CXX_HAS_WARNING-Wunused
-- Performing Test CXX_HAS_WARNING-Wunused - Success
-- Performing Test CXX_HAS_WARNING-Wwrite-strings
-- Performing Test CXX_HAS_WARNING-Wwrite-strings - Success
-- Performing Test CXX_HAS_WARNING-funit-at-a-time
-- Performing Test CXX_HAS_WARNING-funit-at-a-time - Success
-- Performing Test CXX_HAS_WARNING-Wno-strict-overflow
-- Performing Test CXX_HAS_WARNING-Wno-strict-overflow - Success
-- Performing Test CXX_HAS_WARNING-Wno-deprecated
-- Performing Test CXX_HAS_WARNING-Wno-deprecated - Success
-- Performing Test CXX_HAS_WARNING-Wno-invalid-offsetof
-- Performing Test CXX_HAS_WARNING-Wno-invalid-offsetof - Success
-- Performing Test CXX_HAS_WARNING-Woverloaded-virtual
-- Performing Test CXX_HAS_WARNING-Woverloaded-virtual - Success
-- Performing Test CXX_HAS_WARNING-Wstrict-null-sentinel
-- Performing Test CXX_HAS_WARNING-Wstrict-null-sentinel - Success
-- Performing Test have_gold
-- Performing Test have_gold - Success
-- Performing Test CXX_HAS_DISABLE_OPTIMIZATION_FLAG
-- Performing Test CXX_HAS_DISABLE_OPTIMIZATION_FLAG - Failed
--  The remote module: https://github.com/Slicer/itkMGHImageIO.git is cloned into the directory /opt/builds/ants-build/ITKv4/Modules/Remote/MGHIO
-- Enabled ITKAnisotropicSmoothing, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKAntiAlias, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKBiasCorrection, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKBinaryMathematicalMorphology, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKBioCell, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKClassifiers, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKColormap, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKCommon, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKConnectedComponents, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKConvolution, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKCurvatureFlow, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKDICOMParser, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKDeconvolution, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKDeformableMesh, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKDenoising, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKDeprecated, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKDiffusionTensorImage, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKDisplacementField, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKDistanceMap, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKDoubleConversion, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKEigen, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKExpat, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKFEM, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKFEMRegistration, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKFFT, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKFastMarching, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKFiniteDifference, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKGDCM, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKGIFTI, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKGPUAnisotropicSmoothing, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKGPUCommon, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKGPUFiniteDifference, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKGPUImageFilterBase, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKGPUPDEDeformableRegistration, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKGPURegistrationCommon, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKGPUSmoothing, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKGPUThresholding, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKHDF5, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOBMP, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOBioRad, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOCSV, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOGDCM, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOGE, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOGIPL, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOHDF5, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOIPL, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOImageBase, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOJPEG, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOLSM, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOMRC, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOMesh, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOMeta, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIONIFTI, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIONRRD, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOPNG, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIORAW, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOSiemens, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOSpatialObjects, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOStimulate, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOTIFF, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOTransformBase, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOTransformHDF5, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOTransformInsightLegacy, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOTransformMatlab, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOVTK, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIOXML, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageAdaptors, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageCompare, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageCompose, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageFeature, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageFilterBase, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageFunction, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageFusion, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageGradient, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageGrid, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageIntensity, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageLabel, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageNoise, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageSources, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKImageStatistics, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKIntegratedTest, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKJPEG, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKKLMRegionGrowing, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKKWIML, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKKWSys, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKLabelMap, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKLabelVoting, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKLevelSets, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKLevelSetsv4, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKMarkovRandomFieldsClassifiers, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKMathematicalMorphology, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKMesh, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKMetaIO, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKMetricsv4, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKNIFTI, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKNarrowBand, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKNetlib, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKNeuralNetworks, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKNrrdIO, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKOpenJPEG, needed by [ITKReview].
-- Enabled ITKOptimizers, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKOptimizersv4, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKPDEDeformableRegistration, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKPNG, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKPath, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKPolynomials, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKQuadEdgeMesh, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKQuadEdgeMeshFiltering, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKRegionGrowing, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKRegistrationCommon, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKRegistrationMethodsv4, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKReview, requested by Module_ITKReview.
-- Enabled ITKSignedDistanceFunction, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKSmoothing, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKSpatialFunction, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKSpatialObjects, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKStatistics, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKTIFF, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKTestKernel, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKThresholding, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKTransform, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKV3Compatibility, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKVNL, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKVNLInstantiation, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKVTK, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKVideoCore, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKVideoFiltering, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKVideoIO, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKVoronoi, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKVtkGlue, requested by Module_ITKVtkGlue.
-- Enabled ITKWatersheds, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled ITKZLIB, requested by ITK_BUILD_DEFAULT_MODULES.
-- Enabled MGHIO, requested by Module_MGHIO.
-- Double correction test returned: 89255.0/1e22 = 8.9255e-18
-- Checking whether header cstdio is available
-- Checking whether header cstdio is available - yes
-- Checking for Large File Support
-- Checking for Large File Support - yes
-- Checking whether C++ compiler has 'long long'
-- Checking whether C++ compiler has 'long long' - yes
-- Checking whether C++ compiler has '__int64'
-- Checking whether C++ compiler has '__int64' - no
-- Checking whether wstring is available
-- Checking whether wstring is available - yes
-- Checking whether C compiler has ptrdiff_t in stddef.h
-- Checking whether C compiler has ptrdiff_t in stddef.h - yes
-- Checking whether C compiler has ssize_t in unistd.h
-- Checking whether C compiler has ssize_t in unistd.h - yes
-- Checking whether CXX compiler has setenv
-- Checking whether CXX compiler has setenv - yes
-- Checking whether CXX compiler has unsetenv
-- Checking whether CXX compiler has unsetenv - yes
-- Checking whether CXX compiler has environ in stdlib.h
-- Checking whether CXX compiler has environ in stdlib.h - no
-- Checking whether CXX compiler has utimes
-- Checking whether CXX compiler has utimes - yes
-- Checking whether CXX compiler has utimensat
-- Checking whether CXX compiler has utimensat - yes
-- Checking whether CXX compiler struct stat has st_mtim member
-- Checking whether CXX compiler struct stat has st_mtim member - yes
-- Checking whether CXX compiler struct stat has st_mtimespec member
-- Checking whether CXX compiler struct stat has st_mtimespec member - no
-- Looking for include files sys/types.h, ifaddrs.h
-- Looking for include files sys/types.h, ifaddrs.h - found
-- Checking whether CXX compiler has rlimit64
-- Checking whether CXX compiler has rlimit64 - yes
-- Checking whether CXX compiler has atol
-- Checking whether CXX compiler has atol - yes
-- Checking whether CXX compiler has atoll
-- Checking whether CXX compiler has atoll - yes
-- Checking whether CXX compiler has _atoi64
-- Checking whether CXX compiler has _atoi64 - no
-- Looking for C++ include execinfo.h
-- Looking for C++ include execinfo.h - found
-- Checking whether backtrace works with this C++ compiler
-- Checking whether backtrace works with this C++ compiler - yes
-- Looking for C++ include dlfcn.h
-- Looking for C++ include dlfcn.h - found
-- Checking whether dladdr works with this C++ compiler
-- Checking whether dladdr works with this C++ compiler - yes
-- Looking for C++ include cxxabi.h
-- Looking for C++ include cxxabi.h - found
-- Checking whether cxxabi works with this C++ compiler
-- Checking whether cxxabi works with this C++ compiler - yes
-- Checking whether CXX compiler has getloadavg
-- Checking whether CXX compiler has getloadavg - yes
-- Performing Test VCL_HAS_BOOL
-- Performing Test VCL_HAS_BOOL - Success
-- Performing Test VCL_HAS_LONG_LONG
-- Performing Test VCL_HAS_LONG_LONG - Success
-- Performing Test VCL_HAS_TYPENAME
-- Performing Test VCL_HAS_TYPENAME - Success
-- Performing Test VCL_HAS_EXPORT
-- Performing Test VCL_HAS_EXPORT - Success
-- Performing Test VCL_HAS_MUTABLE
-- Performing Test VCL_HAS_MUTABLE - Success
-- Performing Test VCL_HAS_EXPLICIT
-- Performing Test VCL_HAS_EXPLICIT - Success
-- Performing Test VCL_HAS_DYNAMIC_CAST
-- Performing Test VCL_HAS_DYNAMIC_CAST - Success
-- Performing Test VCL_HAS_RTTI
-- Performing Test VCL_HAS_RTTI - Success
-- Performing Test VCL_DEFAULT_VALUE
-- Performing Test VCL_DEFAULT_VALUE - Success
-- Performing Test VCL_HAS_MEMBER_TEMPLATES
-- Performing Test VCL_HAS_MEMBER_TEMPLATES - Success
-- Performing Test VCL_CAN_DO_PARTIAL_SPECIALIZATION
-- Performing Test VCL_CAN_DO_PARTIAL_SPECIALIZATION - Success
-- Performing Test VCL_ALLOWS_INLINE_INSTANTIATION
-- Performing Test VCL_ALLOWS_INLINE_INSTANTIATION - Success
-- Performing Test VCL_NEEDS_INLINE_INSTANTIATION
-- Performing Test VCL_NEEDS_INLINE_INSTANTIATION - Success
-- Performing Test VCL_CAN_DO_STATIC_TEMPLATE_MEMBER
-- Performing Test VCL_CAN_DO_STATIC_TEMPLATE_MEMBER - Success
-- Performing Test VCL_CAN_DO_NON_TYPE_FUNCTION_TEMPLATE_PARAMETER
-- Performing Test VCL_CAN_DO_NON_TYPE_FUNCTION_TEMPLATE_PARAMETER - Success
-- Performing Test VCL_NEED_FRIEND_FOR_TEMPLATE_OVERLOAD
-- Performing Test VCL_NEED_FRIEND_FOR_TEMPLATE_OVERLOAD - Success
-- Performing Test VCL_OVERLOAD_CAST
-- Performing Test VCL_OVERLOAD_CAST - Failed
-- Performing Test VCL_NULL_TMPL_ARGS
-- Performing Test VCL_NULL_TMPL_ARGS - Failed
-- Performing Test VCL_NO_STATIC_DATA_MEMBERS
-- Performing Test VCL_NO_STATIC_DATA_MEMBERS - Success
-- Performing Test VCL_HAS_TEMPLATE_SYMBOLS
-- Performing Try-Run Test VCL_HAS_TEMPLATE_SYMBOLS - Test Compilation Failed
-- Performing Test VCL_CAN_DO_COMPLETE_DEFAULT_TYPE_PARAMETER
-- Performing Test VCL_CAN_DO_COMPLETE_DEFAULT_TYPE_PARAMETER - Success
-- Performing Test VCL_CAN_DO_TEMPLATE_DEFAULT_TYPE_PARAMETER
-- Performing Test VCL_CAN_DO_TEMPLATE_DEFAULT_TYPE_PARAMETER - Success
-- Performing Test VCL_TEMPLATE_MATCHES_TOO_OFTEN
-- Performing Test VCL_TEMPLATE_MATCHES_TOO_OFTEN - Success
-- Performing Test VCL_HAS_EXCEPTIONS
-- Performing Test VCL_HAS_EXCEPTIONS - Success
-- Performing Test VCL_HAS_NAMESPACES
-- Performing Test VCL_HAS_NAMESPACES - Success
-- Performing Test VCL_ALLOWS_NAMESPACE_STD
-- Performing Test VCL_ALLOWS_NAMESPACE_STD - Success
-- Performing Test VCL_NEEDS_NAMESPACE_STD
-- Performing Test VCL_NEEDS_NAMESPACE_STD - Failed
-- Performing Test VCL_HAS_SLICED_DESTRUCTOR_BUG
-- Performing Test VCL_HAS_SLICED_DESTRUCTOR_BUG - Success
-- Performing Test VCL_HAS_WORKING_STRINGSTREAM
-- Performing Test VCL_HAS_WORKING_STRINGSTREAM - Success
-- Performing Test VXL_UNISTD_USLEEP_IS_VOID
-- Performing Test VXL_UNISTD_USLEEP_IS_VOID - Success
-- Performing Test VXL_STDLIB_HAS_QSORT
-- Performing Test VXL_STDLIB_HAS_QSORT - Success
-- Performing Test VXL_C_MATH_HAS_LROUND
-- Performing Test VXL_C_MATH_HAS_LROUND - Success
-- Performing Test VCL_HAS_LFS
-- Performing Test VCL_HAS_LFS - Success
-- Performing Test VXL_HAS_DBGHELP_H
-- Performing Test VXL_HAS_DBGHELP_H - Failed
-- Performing Test VCL_DEFINE_SPECIALIZATION
-- Performing Test VCL_DEFINE_SPECIALIZATION - Success
-- Performing Test VCL_CANNOT_SPECIALIZE_CV
-- Performing Test VCL_CANNOT_SPECIALIZE_CV - Success
-- Performing Test VXL_PIC_COMPATIBLE
-- Performing Test VXL_PIC_COMPATIBLE - Success
-- Looking for C++ include cassert
-- Looking for C++ include cassert - found
-- Looking for C++ include ciso646
-- Looking for C++ include ciso646 - found
-- Looking for C++ include csetjmp
-- Looking for C++ include csetjmp - found
-- Looking for C++ include cstdio
-- Looking for C++ include cstdio - found
-- Looking for C++ include ctime
-- Looking for C++ include ctime - found
-- Looking for C++ include cctype
-- Looking for C++ include cctype - found
-- Looking for C++ include climits
-- Looking for C++ include climits - found
-- Looking for C++ include csignal
-- Looking for C++ include csignal - found
-- Looking for C++ include cstdlib
-- Looking for C++ include cstdlib - found
-- Looking for C++ include cwchar
-- Looking for C++ include cwchar - found
-- Looking for C++ include cerrno
-- Looking for C++ include cerrno - found
-- Looking for C++ include clocale
-- Looking for C++ include clocale - found
-- Looking for C++ include cstdarg
-- Looking for C++ include cstdarg - found
-- Looking for C++ include cstring
-- Looking for C++ include cstring - found
-- Looking for C++ include cwctype
-- Looking for C++ include cwctype - found
-- Looking for C++ include cfloat
-- Looking for C++ include cfloat - found
-- Looking for C++ include cmath
-- Looking for C++ include cmath - found
-- Looking for C++ include cstddef
-- Looking for C++ include cstddef - found
-- Looking for C++ include algorithm
-- Looking for C++ include algorithm - found
-- Looking for C++ include iomanip
-- Looking for C++ include iomanip - found
-- Looking for C++ include list
-- Looking for C++ include list - found
-- Looking for C++ include ostream
-- Looking for C++ include ostream - found
-- Looking for C++ include streambuf
-- Looking for C++ include streambuf - found
-- Looking for C++ include bitset
-- Looking for C++ include bitset - found
-- Looking for C++ include ios
-- Looking for C++ include ios - found
-- Looking for C++ include locale
-- Looking for C++ include locale - found
-- Looking for C++ include queue
-- Looking for C++ include queue - found
-- Looking for C++ include string
-- Looking for C++ include string - found
-- Looking for C++ include complex
-- Looking for C++ include complex - found
-- Looking for C++ include iosfwd
-- Looking for C++ include iosfwd - found
-- Looking for C++ include map
-- Looking for C++ include map - found
-- Looking for C++ include set
-- Looking for C++ include set - found
-- Looking for C++ include typeinfo
-- Looking for C++ include typeinfo - found
-- Looking for C++ include deque
-- Looking for C++ include deque - found
-- Looking for C++ include iostream
-- Looking for C++ include iostream - found
-- Looking for C++ include memory
-- Looking for C++ include memory - found
-- Looking for C++ include sstream
-- Looking for C++ include sstream - found
-- Looking for C++ include utility
-- Looking for C++ include utility - found
-- Looking for C++ include exception
-- Looking for C++ include exception - found
-- Looking for C++ include istream
-- Looking for C++ include istream - found
-- Looking for C++ include new
-- Looking for C++ include new - found
-- Looking for C++ include stack
-- Looking for C++ include stack - found
-- Looking for C++ include valarray
-- Looking for C++ include valarray - found
-- Looking for C++ include fstream
-- Looking for C++ include fstream - found
-- Looking for C++ include iterator
-- Looking for C++ include iterator - found
-- Looking for C++ include numeric
-- Looking for C++ include numeric - found
-- Looking for C++ include stdexcept
-- Looking for C++ include stdexcept - found
-- Looking for C++ include vector
-- Looking for C++ include vector - found
-- Looking for C++ include functional
-- Looking for C++ include functional - found
-- Looking for C++ include strstream
-- Looking for C++ include strstream - found
-- Looking for C++ include iso646.h
-- Looking for C++ include iso646.h - found
-- Looking for C++ include emmintrin.h
-- Looking for C++ include emmintrin.h - found
-- Looking for C++ include windows.h
-- Looking for C++ include windows.h - not found
-- Looking for include file assert.h
-- Looking for include file assert.h - found
-- Looking for include file pthread.h
-- Looking for include file pthread.h - found
-- Looking for include file semaphore.h
-- Looking for include file semaphore.h - found
-- Looking for include file dlfcn.h
-- Looking for include file dlfcn.h - found
-- Looking for include file fcntl.h
-- Looking for include file fcntl.h - found
-- Looking for include file inttypes.h
-- Looking for include file inttypes.h - found
-- Looking for include file limits.h
-- Looking for include file limits.h - found
-- Looking for include file malloc.h
-- Looking for include file malloc.h - found
-- Looking for include file memory.h
-- Looking for include file memory.h - found
-- Looking for include file search.h
-- Looking for include file search.h - found
-- Looking for include file stdint.h
-- Looking for include file stdint.h - found
-- Looking for include file stdlib.h
-- Looking for include file stdlib.h - found
-- Looking for include file strings.h
-- Looking for include file strings.h - found
-- Looking for include file string.h
-- Looking for include file string.h - found
-- Looking for include file sys/stat.h
-- Looking for include file sys/stat.h - found
-- Looking for include file sys/times.h
-- Looking for include file sys/times.h - found
-- Looking for include file sys/types.h
-- Looking for include file sys/types.h - found
-- Looking for include file unistd.h
-- Looking for include file unistd.h - found
-- Looking for C++ include tr1/memory
-- Looking for C++ include tr1/memory - found
-- Looking for std::shared_ptr<void>
-- Looking for std::shared_ptr<void> - not found.
-- Looking for std::tr1::shared_ptr<void>
-- Looking for std::tr1::shared_ptr<void> - not found.
-- Looking for std::tr1::shared_ptr<void>
-- Looking for std::tr1::shared_ptr<void> - found
-- Performing Test VXL_HAS_SSE2_HARDWARE_SUPPORT
-- Performing Test VXL_HAS_SSE2_HARDWARE_SUPPORT - Success
-- Performing Test VXL_HAS_MM_MALLOC
-- Performing Test VXL_HAS_MM_MALLOC - Success
-- Performing Test VXL_HAS_ALIGNED_MALLOC
-- Performing Test VXL_HAS_ALIGNED_MALLOC - Failed
-- Performing Test VXL_HAS_MINGW_ALIGNED_MALLOC
-- Performing Test VXL_HAS_MINGW_ALIGNED_MALLOC - Failed
-- Performing Test VXL_HAS_POSIX_MEMALIGN
-- Performing Test VXL_HAS_POSIX_MEMALIGN - Failed
-- Performing Test VCL_COMPLEX_POW_WORKS
-- Performing Test VCL_COMPLEX_POW_WORKS - Success
-- Performing Test VXL_HAS_STD_ISNAN
-- Performing Test VXL_HAS_STD_ISNAN - Success
-- Performing Test VXL_HAS_STD_ISINF
-- Performing Test VXL_HAS_STD_ISINF - Success
-- Performing Test VXL_HAS_STD_ISFINITE
-- Performing Test VXL_HAS_STD_ISFINITE - Success
-- Performing Test VXL_HAS_STD_ISNORMAL
-- Performing Test VXL_HAS_STD_ISNORMAL - Success
-- Looking for sqrtf
-- Looking for sqrtf - found
-- Looking for finite
-- Looking for finite - found
-- Looking for finitef
-- Looking for finitef - found
-- Looking for finitel
-- Looking for finitel - found
-- Looking for finite
-- Looking for finite - not found
-- Looking for lrand48
-- Looking for lrand48 - found
-- Looking for drand48
-- Looking for drand48 - found
-- Looking for srand48
-- Looking for srand48 - found
-- Check if the system is big endian
-- Searching 16 bit integer
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of unsigned short
-- Check size of unsigned short - done
-- Using unsigned short
-- Check if the system is big endian - little endian
-- Performing Test VCL_CAN_DO_IMPLICIT_TEMPLATES
-- Performing Test VCL_CAN_DO_IMPLICIT_TEMPLATES - Success
-- Performing Test VCL_CHAR_IS_SIGNED
-- Performing Test VCL_CHAR_IS_SIGNED - Success
-- Looking for 8-bit int. [Checking char...]
-- Looking for 8-bit int. Found char.
-- Looking for 8-bit int. [Checking short...]
-- Looking for 8-bit int. [Checking char...]
-- Looking for 8-bit int. Found char.
-- Looking for 16-bit int. [Checking char...]
-- Looking for 16-bit int. [Checking int...]
-- Looking for 16-bit int. [Checking short...]
-- Looking for 16-bit int. Found short.
-- Looking for 32-bit int. [Checking short...]
-- Looking for 32-bit int. [Checking long...]
-- Looking for 32-bit int. [Checking int...]
-- Looking for 32-bit int. Found int.
-- Looking for 64-bit int. [Checking __int64...]
-- Looking for 64-bit int. [Checking long long...]
-- Looking for 64-bit int. [Checking long...]
-- Looking for 64-bit int. Found long.
-- Looking for 32-bit float. [Checking long double...]
-- Looking for 32-bit float. [Checking double...]
-- Looking for 32-bit float. [Checking float...]
-- Looking for 32-bit float. Found float.
-- Looking for 64-bit float. [Checking float...]
-- Looking for 64-bit float. [Checking long double...]
-- Looking for 64-bit float. [Checking double...]
-- Looking for 64-bit float. Found double.
-- Looking for 96-bit float. [Checking float...]
-- Looking for 96-bit float. [Checking double...]
-- Looking for 96-bit float. [Checking long double...]
-- Looking for 96-bit float. Not found.
-- Looking for 128-bit float. [Checking float...]
-- Looking for 128-bit float. [Checking double...]
-- Looking for 128-bit float. [Checking long double...]
-- Looking for 128-bit float. Found long double.
-- Looking for C++ include unistd.h
-- Looking for C++ include unistd.h - found
-- Looking for useconds_t
-- Looking for useconds_t - found
-- Looking for intptr_t
-- Looking for intptr_t - found
-- Looking for ualarm
-- Looking for ualarm - found
-- Looking for usleep
-- Looking for usleep - found
-- Looking for lchown
-- Looking for lchown - found
-- Looking for pread
-- Looking for pread - found
-- Looking for pwrite
-- Looking for pwrite - found
-- Looking for tell
-- Looking for tell - not found
-- Looking for getpid
-- Looking for getpid - found
-- Looking for gethostname
-- Looking for gethostname - found
-- Looking for C++ include limits
-- Looking for C++ include limits - found
-- Performing Test VCL_NUMERIC_LIMITS_HAS_INFINITY
-- Performing Test VCL_NUMERIC_LIMITS_HAS_INFINITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Success
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
-- Check if the system is big endian
-- Searching 16 bit integer
-- Using unsigned short
-- Check if the system is big endian - little endian
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Checking to see if this platform has the __FUNCTION__ C-Preprocessor directive
-- Checking to see if this platform supports has the __FUNCTION__ C-Preprocessor directive - yes
-- Looking for include file fenv.h
-- Looking for include file fenv.h - found
-- Looking for include file emmintrin.h
-- Looking for include file emmintrin.h - found
-- Performing Test ITK_COMPILER_SUPPORTS_SSE2_64
-- Performing Test ITK_COMPILER_SUPPORTS_SSE2_64 - Success
-- Performing Test ITK_COMPILER_SUPPORTS_SSE2_32
-- Performing Test ITK_COMPILER_SUPPORTS_SSE2_32 - Success
-- Performing Test ITK_COMPILER_DOES_NOT_NEED_MSSE2_FLAG
-- Performing Test ITK_COMPILER_DOES_NOT_NEED_MSSE2_FLAG - Success
-- Checking for builtin __sync_add_and_fetch
-- Checking for builtin __sync_add_and_fetch -- success
-- Looking for C++ include tr1/type_traits
-- Looking for C++ include tr1/type_traits - found
-- Looking for C++ include type_traits
-- Looking for C++ include type_traits - not found
-- Performing Test gcc_has_no_associative_math
-- Performing Test gcc_has_no_associative_math - Success
-- Check for STD namespace
-- Check for STD namespace - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Performing Test _HAS_fno-ipa-cp-clone
-- Performing Test _HAS_fno-ipa-cp-clone - Success
-- Looking for inttypes.h
-- Looking for inttypes.h - found
-- Found ZLIB: itkzlib  
-- Found EXPAT: ITKEXPAT  
-- Looking for include file sys/time.h
-- Looking for include file sys/time.h - found
-- Looking for include files sys/time.h, winsock.h
-- Looking for include files sys/time.h, winsock.h - not found
-- Looking for include files sys/time.h, byteswap.h
-- Looking for include files sys/time.h, byteswap.h - found
-- Performing Test GDCM_HAVE_BYTESWAP_H
-- Performing Test GDCM_HAVE_BYTESWAP_H - Success
-- Looking for rpc.h
-- Looking for rpc.h - not found
-- Looking for langinfo.h
-- Looking for langinfo.h - found
-- Looking for nl_langinfo
-- Looking for nl_langinfo - found
-- Looking for strcasecmp
-- Looking for strcasecmp - found
-- Looking for strncasecmp
-- Looking for strncasecmp - found
-- Looking for snprintf
-- Looking for snprintf - found
-- Looking for strptime
-- Looking for strptime - found
-- Looking for _stricmp
-- Looking for _stricmp - not found
-- Looking for _strnicmp
-- Looking for _strnicmp - not found
-- Looking for _snprintf
-- Looking for _snprintf - not found
-- Looking for lround
-- Looking for lround - found
-- Looking for gettimeofday
-- Looking for gettimeofday - found
-- Performing Test GDCM_CXX_HAS_FUNCTION
-- Performing Test GDCM_CXX_HAS_FUNCTION - Success
-- Performing Test GDCM_HAVE_WCHAR_IFSTREAM
-- Performing Test GDCM_HAVE_WCHAR_IFSTREAM - Failed
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for stdlib.h
-- Looking for stdlib.h - found
-- Check size of int
-- Check size of int - done
-- Check size of long
-- Check size of long - done
-- Check size of void*
-- Check size of void* - done
-- Check size of char
-- Check size of char - done
-- Check size of short
-- Check size of short - done
-- Check size of float
-- Check size of float - done
-- Check size of double
-- Check size of double - done
-- Looking for limits.h
-- Looking for limits.h - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Looking for include files sys/types.h, sys/prctl.h
-- Looking for include files sys/types.h, sys/prctl.h - found
-- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so
-- Looking for XOpenDisplay in /usr/lib/x86_64-linux-gnu/libX11.so;/usr/lib/x86_64-linux-gnu/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Found X11: /usr/lib/x86_64-linux-gnu/libX11.so
-- Looking for 3 include files stdlib.h, ..., inttypes.h
-- Looking for 3 include files stdlib.h, ..., inttypes.h - found
-- Looking for 4 include files stdlib.h, ..., sys/ioctl.h
-- Looking for 4 include files stdlib.h, ..., sys/ioctl.h - found
-- Looking for 5 include files stdlib.h, ..., sys/time.h
-- Looking for 5 include files stdlib.h, ..., sys/time.h - found
-- Looking for 6 include files stdlib.h, ..., sys/file.h
-- Looking for 6 include files stdlib.h, ..., sys/file.h - found
-- Looking for 7 include files stdlib.h, ..., sys/socket.h
-- Looking for 7 include files stdlib.h, ..., sys/socket.h - found
-- Looking for 8 include files stdlib.h, ..., sys/sockio.h
-- Looking for 8 include files stdlib.h, ..., sys/sockio.h - not found
-- Looking for 8 include files stdlib.h, ..., net/if.h
-- Looking for 8 include files stdlib.h, ..., net/if.h - found
-- Looking for 9 include files stdlib.h, ..., netinet/in.h
-- Looking for 9 include files stdlib.h, ..., netinet/in.h - found
-- Looking for 10 include files stdlib.h, ..., net/if_dl.h
-- Looking for 10 include files stdlib.h, ..., net/if_dl.h - not found
-- Looking for 10 include files stdlib.h, ..., net/if_arp.h
-- Looking for 10 include files stdlib.h, ..., net/if_arp.h - found
-- Looking for strsignal
-- Looking for strsignal - found
-- Looking for ceil in m;
-- Looking for ceil in m; - found
-- Looking for dlopen in dl;m
-- Looking for dlopen in dl;m - found
-- Looking for WSAStartup in ws2_32;m;dl
-- Looking for WSAStartup in ws2_32;m;dl - not found
-- Looking for gethostbyname in wsock32;m;dl
-- Looking for gethostbyname in wsock32;m;dl - not found
-- Looking for gethostname in ucb;m;dl
-- Looking for gethostname in ucb;m;dl - not found
-- Check if the system is big endian
-- Searching 16 bit integer
-- Using unsigned short
-- Check if the system is big endian - little endian
-- Performing Other Test STDC_HEADERS - Success
-- Looking for include file sys/resource.h
-- Looking for include file sys/resource.h - found
-- Looking for include files sys/resource.h, sys/time.h
-- Looking for include files sys/resource.h, sys/time.h - found
-- Looking for 3 include files sys/resource.h, ..., unistd.h
-- Looking for 3 include files sys/resource.h, ..., unistd.h - found
-- Looking for 4 include files sys/resource.h, ..., sys/ioctl.h
-- Looking for 4 include files sys/resource.h, ..., sys/ioctl.h - found
-- Looking for 5 include files sys/resource.h, ..., sys/stat.h
-- Looking for 5 include files sys/resource.h, ..., sys/stat.h - found
-- Looking for 6 include files sys/resource.h, ..., sys/socket.h
-- Looking for 6 include files sys/resource.h, ..., sys/socket.h - found
-- Looking for 7 include files sys/resource.h, ..., sys/types.h
-- Looking for 7 include files sys/resource.h, ..., sys/types.h - found
-- Looking for 8 include files sys/resource.h, ..., stddef.h
-- Looking for 8 include files sys/resource.h, ..., stddef.h - found
-- Looking for 9 include files sys/resource.h, ..., setjmp.h
-- Looking for 9 include files sys/resource.h, ..., setjmp.h - found
-- Looking for 10 include files sys/resource.h, ..., features.h
-- Looking for 10 include files sys/resource.h, ..., features.h - found
-- Looking for 11 include files sys/resource.h, ..., dirent.h
-- Looking for 11 include files sys/resource.h, ..., dirent.h - found
-- Looking for 12 include files sys/resource.h, ..., stdint.h
-- Looking for 12 include files sys/resource.h, ..., stdint.h - found
-- Looking for C++ include stdint.h
-- Looking for C++ include stdint.h - found
-- Looking for 13 include files sys/resource.h, ..., mach/mach_time.h
-- Looking for 13 include files sys/resource.h, ..., mach/mach_time.h - not found
-- Looking for 13 include files sys/resource.h, ..., io.h
-- Looking for 13 include files sys/resource.h, ..., io.h - not found
-- Looking for 13 include files sys/resource.h, ..., winsock2.h
-- Looking for 13 include files sys/resource.h, ..., winsock2.h - not found
-- Looking for 13 include files sys/resource.h, ..., sys/timeb.h
-- Looking for 13 include files sys/resource.h, ..., sys/timeb.h - found
-- Looking for 14 include files sys/resource.h, ..., globus/common.h
-- Looking for 14 include files sys/resource.h, ..., globus/common.h - not found
-- Looking for 14 include files sys/resource.h, ..., pdb.h
-- Looking for 14 include files sys/resource.h, ..., pdb.h - not found
-- Looking for 14 include files sys/resource.h, ..., pthread.h
-- Looking for 14 include files sys/resource.h, ..., pthread.h - found
-- Looking for 15 include files sys/resource.h, ..., srbclient.h
-- Looking for 15 include files sys/resource.h, ..., srbclient.h - not found
-- Looking for 15 include files sys/resource.h, ..., string.h
-- Looking for 15 include files sys/resource.h, ..., string.h - found
-- Looking for 16 include files sys/resource.h, ..., strings.h
-- Looking for 16 include files sys/resource.h, ..., strings.h - found
-- Looking for 17 include files sys/resource.h, ..., stdlib.h
-- Looking for 17 include files sys/resource.h, ..., stdlib.h - found
-- Looking for 18 include files sys/resource.h, ..., memory.h
-- Looking for 18 include files sys/resource.h, ..., memory.h - found
-- Looking for 19 include files sys/resource.h, ..., dlfcn.h
-- Looking for 19 include files sys/resource.h, ..., dlfcn.h - found
-- Looking for 20 include files sys/resource.h, ..., inttypes.h
-- Looking for 20 include files sys/resource.h, ..., inttypes.h - found
-- Looking for 21 include files sys/resource.h, ..., netinet/in.h
-- Looking for 21 include files sys/resource.h, ..., netinet/in.h - found
-- Performing TEST_LFS_WORKS... yes
-- Performing Other Test HAVE_OFF64_T - Success
-- Looking for lseek64
-- Looking for lseek64 - found
-- Looking for fseeko64
-- Looking for fseeko64 - found
-- Looking for ftello64
-- Looking for ftello64 - found
-- Looking for ftruncate64
-- Looking for ftruncate64 - found
-- Looking for fseeko
-- Looking for fseeko - found
-- Looking for ftello
-- Looking for ftello - found
-- Performing Other Test HAVE_STAT64_STRUCT - Success
-- Looking for fstat64
-- Looking for fstat64 - found
-- Looking for stat64
-- Looking for stat64 - found
-- Check size of char
-- Check size of char - done
-- Check size of short
-- Check size of short - done
-- Check size of int
-- Check size of int - done
-- Check size of unsigned
-- Check size of unsigned - done
-- Check size of long
-- Check size of long - done
-- Check size of long long
-- Check size of long long - done
-- Check size of __int64
-- Check size of __int64 - failed
-- Check size of float
-- Check size of float - done
-- Check size of double
-- Check size of double - done
-- Check size of long double
-- Check size of long double - done
-- Check size of int8_t
-- Check size of int8_t - done
-- Check size of uint8_t
-- Check size of uint8_t - done
-- Check size of int_least8_t
-- Check size of int_least8_t - done
-- Check size of uint_least8_t
-- Check size of uint_least8_t - done
-- Check size of int_fast8_t
-- Check size of int_fast8_t - done
-- Check size of uint_fast8_t
-- Check size of uint_fast8_t - done
-- Check size of int16_t
-- Check size of int16_t - done
-- Check size of uint16_t
-- Check size of uint16_t - done
-- Check size of int_least16_t
-- Check size of int_least16_t - done
-- Check size of uint_least16_t
-- Check size of uint_least16_t - done
-- Check size of int_fast16_t
-- Check size of int_fast16_t - done
-- Check size of uint_fast16_t
-- Check size of uint_fast16_t - done
-- Check size of int32_t
-- Check size of int32_t - done
-- Check size of uint32_t
-- Check size of uint32_t - done
-- Check size of int_least32_t
-- Check size of int_least32_t - done
-- Check size of uint_least32_t
-- Check size of uint_least32_t - done
-- Check size of int_fast32_t
-- Check size of int_fast32_t - done
-- Check size of uint_fast32_t
-- Check size of uint_fast32_t - done
-- Check size of int64_t
-- Check size of int64_t - done
-- Check size of uint64_t
-- Check size of uint64_t - done
-- Check size of int_least64_t
-- Check size of int_least64_t - done
-- Check size of uint_least64_t
-- Check size of uint_least64_t - done
-- Check size of int_fast64_t
-- Check size of int_fast64_t - done
-- Check size of uint_fast64_t
-- Check size of uint_fast64_t - done
-- Check size of size_t
-- Check size of size_t - done
-- Check size of ssize_t
-- Check size of ssize_t - done
-- Check size of ptrdiff_t
-- Check size of ptrdiff_t - done
-- Check size of off_t
-- Check size of off_t - done
-- Check size of off64_t
-- Check size of off64_t - done
-- Looking for 22 include files sys/resource.h, ..., stdbool.h
-- Looking for 22 include files sys/resource.h, ..., stdbool.h - found
-- Check size of _Bool
-- Check size of _Bool - done
-- Performing Other Test DEV_T_IS_SCALAR - Success
-- Looking for CLOCK_MONOTONIC
-- Looking for CLOCK_MONOTONIC - not found
-- Looking for gettimeofday
-- Looking for gettimeofday - found
-- Performing Other Test HAVE_TM_GMTOFF - Success
-- Performing Other Test HAVE___TM_GMTOFF - Failed
-- Performing Other Test HAVE_STRUCT_TIMEZONE - Success
-- Performing Other Test GETTIMEOFDAY_GIVES_TZ - Success
-- Performing Other Test TIME_WITH_SYS_TIME - Success
-- Performing Other Test HAVE_TM_ZONE - Failed
-- Performing Other Test HAVE_STRUCT_TM_TM_ZONE - Failed
-- Performing Other Test HAVE_TIMEZONE - Success
-- Performing Other Test HAVE_STAT_ST_BLOCKS - Success
-- Looking for ioctl
-- Looking for ioctl - found
-- Performing Other Test HAVE_STRUCT_VIDEOCONFIG - Failed
-- Performing Other Test HAVE_STRUCT_TEXT_INFO - Failed
-- Looking for _getvideoconfig
-- Looking for _getvideoconfig - not found
-- Looking for gettextinfo
-- Looking for gettextinfo - not found
-- Looking for _scrsize
-- Looking for _scrsize - not found
-- Looking for GetConsoleScreenBufferInfo
-- Looking for GetConsoleScreenBufferInfo - not found
-- Looking for TIOCGWINSZ
-- Looking for TIOCGWINSZ - found
-- Looking for TIOCGETD
-- Looking for TIOCGETD - found
-- Looking for alarm
-- Looking for alarm - found
-- Looking for fork
-- Looking for fork - found
-- Looking for frexpf
-- Looking for frexpf - found
-- Looking for frexpl
-- Looking for frexpl - found
-- Looking for gethostname
-- Looking for gethostname - found
-- Looking for getpwuid
-- Looking for getpwuid - found
-- Looking for getrusage
-- Looking for getrusage - found
-- Looking for lstat
-- Looking for lstat - found
-- Looking for rand_r
-- Looking for rand_r - found
-- Looking for random
-- Looking for random - found
-- Looking for setsysinfo
-- Looking for setsysinfo - not found
-- Looking for signal
-- Looking for signal - found
-- Looking for longjmp
-- Looking for longjmp - found
-- Looking for setjmp
-- Looking for setjmp - found
-- Looking for siglongjmp
-- Looking for siglongjmp - found
-- Looking for sigsetjmp
-- Looking for sigsetjmp - not found
-- Looking for sigprocmask
-- Looking for sigprocmask - found
-- Looking for snprintf
-- Looking for snprintf - found
-- Looking for srandom
-- Looking for srandom - found
-- Looking for strdup
-- Looking for strdup - found
-- Looking for symlink
-- Looking for symlink - found
-- Looking for system
-- Looking for system - found
-- Looking for tmpfile
-- Looking for tmpfile - found
-- Looking for asprintf
-- Looking for asprintf - found
-- Looking for vasprintf
-- Looking for vasprintf - found
-- Looking for waitpid
-- Looking for waitpid - found
-- Looking for vsnprintf
-- Looking for vsnprintf - found
-- Performing Other Test VSNPRINTF_WORKS - Success
-- Looking for sigsetjmp
-- Looking for sigsetjmp - found
-- Performing Other Test HAVE_ATTRIBUTE - Success
-- Performing Other Test HAVE_C99_FUNC - Success
-- Performing Other Test HAVE_FUNCTION - Success
-- Performing Other Test HAVE_C99_DESIGNATED_INITIALIZER - Success
-- Performing Other Test SYSTEM_SCOPE_THREADS - Success
-- Performing Other Test HAVE_SOCKLEN_T - Success
-- Performing Other Test CXX_HAVE_OFFSETOF - Success
-- Performing CXX Test OLD_HEADER_FILENAME - Failed
-- Performing CXX Test H5_NO_NAMESPACE - Failed
-- Performing CXX Test H5_NO_STD - Failed
-- Performing CXX Test BOOL_NOTDEFINED - Failed
-- Performing CXX Test NO_STATIC_CAST - Failed
-- Performing Other Test HAVE_INLINE - Success
-- Performing Other Test HAVE___INLINE__ - Success
-- Performing Other Test HAVE___INLINE - Success
-- Checking for appropriate format for 64 bit long:
-- Checking for appropriate format for 64 bit long: found "l"
-- Looking for difftime
-- Looking for difftime - found
-- Looking for clock_gettime
-- Looking for clock_gettime - found
-- Looking for clock_gettime in rt
-- Looking for clock_gettime in rt - found
-- Looking for clock_gettime in posix4
-- Looking for clock_gettime in posix4 - not found
-- Performing Other Test HAVE_DIRECT - Success
-- Checking IF your system converts long double to (unsigned) long values with special algorithm... no
-- Checking IF your system can convert (unsigned) long to long double values with special algorithm... no
-- Checking IF correctly converting long double to (unsigned) long long values... yes
-- Checking IF correctly converting (unsigned) long long to long double values... yes
-- Checking IF alignment restrictions are strictly enforced... yes
-- Filter ZLIB is ON
-- Looking for include file assert.h
-- Looking for include file assert.h - found
-- Looking for include file dlfcn.h
-- Looking for include file dlfcn.h - found
-- Looking for include file fcntl.h
-- Looking for include file fcntl.h - found
-- Looking for include file limits.h
-- Looking for include file limits.h - found
-- Looking for include file malloc.h
-- Looking for include file malloc.h - found
-- Looking for include file memory.h
-- Looking for include file memory.h - found
-- Looking for include file string.h
-- Looking for include file string.h - found
-- Looking for include file strings.h
-- Looking for include file strings.h - found
-- Looking for include file sys/stat.h
-- Looking for include file sys/stat.h - found
-- Looking for include file windows.h
-- Looking for include file windows.h - not found
-- Looking for include file ieeefp.h
-- Looking for include file ieeefp.h - not found
-- Looking for include file search.h
-- Looking for include file search.h - found
-- Looking for floor
-- Looking for floor - not found
-- Looking for getopt
-- Looking for getopt - found
-- Looking for isascii
-- Looking for isascii - found
-- Looking for memmove
-- Looking for memmove - found
-- Looking for memset
-- Looking for memset - found
-- Looking for mmap
-- Looking for mmap - found
-- Looking for pow
-- Looking for pow - not found
-- Looking for sqrt
-- Looking for sqrt - not found
-- Looking for strcasecmp
-- Looking for strcasecmp - found
-- Looking for strchr
-- Looking for strchr - found
-- Looking for strrchr
-- Looking for strrchr - found
-- Looking for strstr
-- Looking for strstr - found
-- Looking for strtol
-- Looking for strtol - found
-- Looking for strtoul
-- Looking for strtoul - found
-- Check size of int
-- Check size of int - done
-- Check size of long
-- Check size of long - done
-- Check if the system is big endian
-- Searching 16 bit integer
-- Using unsigned short
-- Check if the system is big endian - little endian
-- Check the value of the 22nd bit of a 32-bit quiet-NaN
-- Check the value of the 22nd bit of a 32-bit quiet-NaN - 1
CMake Error at Modules/Bridge/VtkGlue/itk-module-init.cmake:9 (find_package):
  Could not find a package configuration file provided by "VTK" with any of
  the following names:

    VTKConfig.cmake
    vtk-config.cmake

  Add the installation prefix of "VTK" to CMAKE_PREFIX_PATH or set "VTK_DIR"
  to a directory containing one of the above files.  If "VTK" provides a
  separate development package or SDK, be sure it has been installed.
Call Stack (most recent call first):
  CMake/ITKModuleEnablement.cmake:316 (include)
  CMakeLists.txt:317 (include)


-- Configuring incomplete, errors occurred!
See also "/opt/builds/ants-build/ITKv4-build/CMakeFiles/CMakeOutput.log".
See also "/opt/builds/ants-build/ITKv4-build/CMakeFiles/CMakeError.log".
make[2]: *** [ITKv4-prefix/src/ITKv4-stamp/ITKv4-configure] Error 1
CMakeFiles/ITKv4.dir/build.make:106: recipe for target 'ITKv4-prefix/src/ITKv4-stamp/ITKv4-configure' failed
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/ITKv4.dir/all' failed
make[1]: *** [CMakeFiles/ITKv4.dir/all] Error 2
Makefile:116: recipe for target 'all' failed
make: *** [all] Error 2
The command '/bin/sh -c apt-get update &&     apt-get -y upgrade &&     apt-get -y install gcc g++ cmake git zlib1g-dev freeglut3-dev &&     mkdir -p ${SOFTDIR} ${BUILDDIR} && 	cd ${BUILDDIR} &&     git clone ${ANTS_GIT} ants &&     cd ants &&     if [ ${ANTS_VERSION} == 'dev' ]; then git checkout $ANTS_COMMIT_HASH; else git checkout tags/v$ANTS_VERSION; fi &&     cd ${BUILDDIR} &&     mkdir ants-build &&     cd ants-build &&     cmake ../ants -DCMAKE_INSTALL_PREFIX=${SOFTDIR}/ants         -DCMAKE_BUILD_TYPE=Release         -DBUILD_SHARED_LIBS=ON         -DUSE_VTK=ON         -DUSE_SYSTEM_VTK=OFF &&     make -j ${N_CPU} &&     mkdir ${SOFTDIR}/ants &&     mv bin lib ${SOFTDIR}/ants/ &&     mv ITKv4-install ${SOFTDIR}/ants/ITKv4 &&     cd ${BUILDDIR} &&     mv ants/Scripts ${SOFTDIR}/ants/scripts &&     rm -rf ants* &&     apt-get -y remove cmake git gcc g++ zlib1g-dev freeglut3-dev &&     apt-get -y autoremove' returned a non-zero code: 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -43,7 +43,7 @@ RUN if [ -z "$ants_version" ]; then \
     && git clone git://github.com/stnava/ANTs.git ants \
     && cd ants \
     && git fetch origin --tags \
-    && git checkout tags/$ants_version \
+    && git checkout $ants_version \
     && mkdir build \
     && cd build \
     && source /opt/rh/devtoolset-3/enable \

--- a/vtk.Dockerfile
+++ b/vtk.Dockerfile
@@ -1,0 +1,69 @@
+# Dynamic Dockerfile that builds ANTs from source with VTK support.
+#
+# Example:
+#
+#   docker build -t ants:2.2.0 --build-arg ants_version=2.2.0 - < vtk.Dockerfile
+
+FROM centos:6@sha256:12f2e9aa55e245664e86bfdf4eb000ddc316b48d9aa63c3c98ba886416868e49 as builder
+
+ARG NPROC=1
+
+WORKDIR /tmp
+RUN curl -fsSLO https://raw.githubusercontent.com/cms-sw/cms-docker/master/slc6-vanilla/RPM-GPG-KEY-cern \
+    && rpm --import RPM-GPG-KEY-cern \
+    && curl -fsSL -o /etc/yum.repos.d/slc6-scl.repo http://linuxsoft.cern.ch/cern/scl/slc6-scl.repo \
+    && yum install -y -q \
+          curl \
+          devtoolset-3-gcc-c++ \
+          git \
+          make \
+          zlib-devel \
+    && yum clean packages \
+    && rm -rf /var/cache/yum/* && rm -rf /tmp/* \
+    && curl -fsSL https://cmake.org/files/v3.12/cmake-3.12.2.tar.gz | tar -xz \
+    && cd cmake-3.12.2 \
+    && source /opt/rh/devtoolset-3/enable \
+    && printf "\n\n+++++++++++++++++++++++++++++++++\n\
+BUILDING CMAKE WITH $NPROC PROCESS(ES)\n\
++++++++++++++++++++++++++++++++++\n\n" \
+    && ./bootstrap --parallel=$NPROC -- -DCMAKE_BUILD_TYPE:STRING=Release \
+    && make -j$NPROC \
+    && make install \
+    && cd .. \
+    && rm -rf *
+
+ARG ants_version
+
+ENV ANTS_VERSION=$ants_version
+WORKDIR /src
+RUN if [ -z "$ants_version" ]; then \
+        echo "ERROR: ants_version not defined" && exit 1; \
+    fi \
+    && echo "Compiling ANTs version $ants_version" \
+    && git clone git://github.com/stnava/ANTs.git ants \
+    && cd ants \
+    && git fetch origin --tags \
+    && git checkout $ants_version \
+    && mkdir build \
+    && cd build \
+    && source /opt/rh/devtoolset-3/enable \
+    && yum install -y mesa-libGL-devel libXt-devel \
+    && printf "\n\n++++++++++++++++++++++++++++++++\n\
+BUILDING ANTS WITH $NPROC PROCESS(ES)\n\
+++++++++++++++++++++++++++++++++\n\n" \
+    && cmake -DUSE_SYSTEM_VTK=OFF -DUSE_VTK=ON .. \
+    && make -j$NPROC \
+    && mkdir -p /opt/ants \
+    && mv bin/* /opt/ants && mv ../Scripts/* /opt/ants \
+    && cd .. \
+    && rm -rf build
+
+FROM centos:7.5.1804
+
+COPY --from=builder /opt/ants /opt/ants
+
+ENV ANTSPATH=/opt/ants/ \
+    PATH=/opt/ants:$PATH
+
+LABEL maintainer="Jakub Kaczmarzyk <jakubk@mit.edu>" \
+      ants_version=$ants_version

--- a/vtk.Dockerfile
+++ b/vtk.Dockerfile
@@ -2,7 +2,7 @@
 #
 # Example:
 #
-#   docker build -t ants:2.2.0 --build-arg ants_version=2.2.0 - < vtk.Dockerfile
+#   docker build -t ants:vtk --build-arg NPROC=8 --build-arg ants_version=master - < vtk.Dockerfile
 
 FROM centos:6@sha256:12f2e9aa55e245664e86bfdf4eb000ddc316b48d9aa63c3c98ba886416868e49 as builder
 
@@ -32,7 +32,7 @@ BUILDING CMAKE WITH $NPROC PROCESS(ES)\n\
     && cd .. \
     && rm -rf *
 
-ARG ants_version
+ARG ants_version="master"
 
 ENV ANTS_VERSION=$ants_version
 WORKDIR /src

--- a/vtk.Dockerfile
+++ b/vtk.Dockerfile
@@ -62,6 +62,9 @@ FROM centos:7.5.1804
 
 COPY --from=builder /opt/ants /opt/ants
 
+RUN yum install -y mesa-libGL libXt \
+    && yum clean all
+
 ENV ANTSPATH=/opt/ants/ \
     PATH=/opt/ants:$PATH
 

